packages/vue_frontend_rpc/test/components/Editor.spec.js

```diff
@@ -0,0 +1,88 @@
+import Editor from "../../src/components/Editor.vue";
+import { expect } from "chai";
+import { mount } from "@vue/test-utils";
```

Comment:
There are manny warnings that come from vuetify 

![image](https://github.com/SAP/task-explorer/assets/9718939/d92254be-8668-4360-ba80-7b2dc44e9b95)

please see Alex global comment to disable these warnings